README.md

```diff
@@ -55,6 +55,21 @@ cd ../boundaries
 for x in $AREAS; do
   wget https://assets.od2net.org/boundaries/$x.geojson
 done
+cd ..
+```
+
+There are two more directories particular to Scotland. To cache all of that data:
```

Comment:
I didn't run this myself due to limited internet, but fairly sure it's at least close to correct. Let me know if you have trouble with it.